src/Contacts/src/CNLabeledValue.m

```diff
@@ -22,35 +22,69 @@
 @implementation CNLabeledValue
 
 - (instancetype)initWithLabel:(NSString *)label value:(id)value {
+    if (self = [super init]) {
+//        [self applyLabeledValueVariablesIdentifier: [[NSUUID alloc] init].UUIDString label:label value:value];
+        [self applyLabeledValueVariablesIdentifier: [[NSUUID alloc] init].UUIDString label:label value:value];
```

Comment:
You're leaking the UUID here. `alloc` returns you a retained object, and you're never releasing it.

Compare:
* `[[NSUUID alloc] init]` — `alloc` retains a retained instance, you own it and have to `release` it later
* `[NSUUID new]` — same
* `[[[NSUUID alloc] init] autorelease]` — properly released
* `[NSUUID UUID]` — returns unretained (as most methods do), and [the implementation](https://github.com/darlinghq/darling-foundation/blob/master/src/NSUUID.m#L36-L39) does `alloc`-`init`-`autorelease`